library/cluster_vm.py

```diff
@@ -67,6 +67,10 @@
         arguments I(name), I(metadata_name)
       - C(define_colocation) Define colocation rules. Require
         arguments I(name), I(colocated_vms)
+      - C(add_pacemaker_remote) Add a pacemaker remote. Require arguments
+        I(name), −I(remote_name), I(remote_address)
+      - -C(remove_pacemaker_remote) Remove a pacemaker remote. Require arguments
+         I(name)
     choices: [ create, remove, list_vms, start, status, stop, clone,
```

Comment:
Should'nt you add "add_pacemaker_remote" and "remove_pacemaker_remote" in the choices ?

Reply:
Done.